src/manifolds/Circle.jl

```diff
@@ -115,6 +115,23 @@ function flat!(::Circle, v::FVector{CotangentSpaceType}, x, w::FVector{TangentSp
 end
 flat(M::Circle, x::Number, w::FVector{TangentSpaceType}) = FVector(CotangentSpace,w.data)
 
+
+function get_coordinates(M::Circle{ℝ}, x, v, B::ArbitraryOrthonormalBasis)
+    return v
+end
+
+function get_coordinates(M::Circle{ℂ}, x, v, B::ArbitraryOrthonormalBasis)
+    return @SVector [imag(v[1] / x[1])]
```

Comment:
This knocks off some compute cycles
```suggestion
    v, x = v[1], x[1]
    w = imag(v) * real(x) - real(v) * imag(x)
    return @SVector [w]
```